apps/prairielearn/src/pages/instructorQuestionSettings/instructorQuestionSettings.html.ts

```diff
@@ -77,261 +79,163 @@ export function InstructorQuestionSettings({
+                    id="qid"
+                    name="qid"
+                    value="${resLocals.question.qid}"
+                    disabled
+                  />
+                  <small class="form-text text-muted">
+                    This is a unique identifier for the question. (e.g., "addNumbers")
+                  </small>
+                </div>
+                <div>
+                  <h2 class="h5">Topic</h2>
+                  <div class="row px-3">
+                    <div class="col-1">
+                      ${renderEjs(__filename, "<%- include('../partials/topic') %>", {
+                        topic: resLocals.topic,
+                      })}
+                    </div>
+                    <div class="col-auto">${resLocals.topic.description}</div>
+                  </div>
+                </div>
+                <hr />
+                <div>
+                  <h2 class="h5">Tags</h2>
+                  ${TagRows({ tags: resLocals.tags })}
+                </div>
+                <hr />
+                <div>
+                  <h2 class="h5">Assessments</h2>
+                  <div>
+                    ${AssessmentRows({
+                      assessmentsWithQuestion,
                     })}
-                  </td>
-                </tr>
-                <tr>
-                  <th>Assessments</th>
-                  <td>
-                    ${resLocals.assessments
-                      ? renderEjs(__filename, "<%- include('../partials/assessments') %>", {
-                          assessments: resLocals.assessments,
-                          urlPrefix: resLocals.urlPrefix,
-                        })
-                      : ''}
-                  </td>
-                </tr>
-                ${sharingEnabled
+                  </div>
+                </div>
+              </form>
+              ${sharingEnabled
+                ? html`
+                    <hr />
+                    <div>
+                      <h2 class="h5">Sharing</h2>
+                      <div data-testid="shared-with">
+                        ${QuestionSharing({
+                          questionSharedPublicly: resLocals.question.shared_publicly,
+                          sharingSetsIn,
+                          hasCoursePermissionOwn: resLocals.authz_data.has_course_permission_own,
+                          sharingSetsOther,
+                          csrfToken: resLocals.__csrf_token,
+                          qid: resLocals.question.qid,
+                        })}
+                      </div>
+                    </div>
+                  `
+                : ''}
+              <hr />
+              ${resLocals.question.type === 'Freeform' &&
+              resLocals.question.grading_method !== 'External' &&
+              resLocals.authz_data.has_course_permission_view
+                ? html`
+                    <div>
+                      <h2 class="h5">Tests</h5>
+                      <div class="pl-3">
```

Comment:
Why add horizontal padding here?

Reply:
I had added horizontal padding here and elsewhere to help indent the information and help with readability. Removed in [381d728](https://github.com/PrairieLearn/PrairieLearn/pull/9687/commits/381d7287d02a906cb1405663be90d4f84dc38348).